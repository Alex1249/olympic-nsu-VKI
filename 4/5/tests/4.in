69 32
806 -604 -287 688 -713 631 -691 -15 938 -442 -198 355 -756 -951 -830 686 17 -729 405 -791 -298 -899 129 -888 -400 -896 -819 984 -767 -320 456 163 326 -666 -227 37 -176 -284 -33 -323 622 845 -868 250 -443 724 608 166 726 -422 -585 533 28 -916 171 257 -627 194 -169 -881 648 265 -895 -336 42 -798 -63 93 828
0 865
0 1688
0 1113
0 2087
0 998
0 2042
0 748
0 86
1 1376
0 1939
0 1657
1 240
0 2121
0 2181
0 13
1 1274
0 1699
1 1910
1 1724
0 1786
0 1138
1 1551
1 956
0 701
0 1120
0 158
0 40
0 1202
0 1048
0 628
0 158
1 1708
