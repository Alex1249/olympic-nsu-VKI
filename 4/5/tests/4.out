806 -8389212 -799 688 -715 631 -691 -15 938 -442 -198 355 -756 -951 -830 686 17 -729 405 -2839 -298 -903 129 -525176 -400 -896 -819 984 -767 -312 456 163 326 -666 -227 37 -176 -8476 -33 -323 622 845 -868 -2147483398 -443 724 608 166 66262 -422 -585 533 28 -268436372 171 257 -627 194 -169 -369 648 265 -895 -368 42 -16778014 -4194367 93 828 
